src/instructlab/sdg/pipelines/full/grounded_skills.yaml

```diff
@@ -10,6 +10,7 @@ blocks:
       temperature: 0.7
       max_tokens: 2048
       n: 10
+      seed: 42
```

Comment:
I think you'll have to update the schema we just merged in #132 to allow this new parameter.  Try running `make validate-pipelines`

Reply:
I did that and looks like it passed. Is there anything else that I need to do here?

Reply:
Sorry, the schema wasn't set properly to catch this.  See this commit - https://github.com/instructlab/sdg/pull/137/commits/94b3c0a3145d6e830a6577a9b0848b6eec5e040e

If `additionalParameters: false` had been set, this change would have failed validation. I fixed that now, though!

Reply:
@aakankshaduggal since I added a commit to the PR, will you review and approve as well that it looks OK to you?